### PR TITLE
Fixed SurfacePlane functionality in Editor

### PR DESCRIPTION
**Pull request by jwittner**
Added method BoundedPlane to build from Transform.
In SurfacePlane Awake, set plane to BoundedPlane from self transform.
In SurfacePlane Start, update the surface plane.

Note that UpdateToSurface is delayed until start so that SurfaceMeshesToPlanes.Instance can be initialized by the it's Awake call. Setting the plane must happen in Awake so that modifications when instantiated dynamically aren't overwritten.

### CODE REVIEW

**Comment by jwittner** [comment]
#34

File: Assets/HoloToolkit/SpatialMapping/Scripts/PlaneFinding.cs [hunk 1]
```diff
@@ -22,6 +22,18 @@ public struct BoundedPlane
         public Plane Plane;
         public OrientedBoundingBox Bounds;
         public float Area;
+
+        public BoundedPlane(Transform xform)
```
Comment (NeerajW) [inline Assets/HoloToolkit/SpatialMapping/Scripts/PlaneFinding.cs]: Please add a summary comment to this function.

**Comment by angelaHillier** [comment]
OK, I tested this change to make sure that it does not impact our other spatial mapping projects/components. Looks good, so merging the pull request now.

**Comment by jwittner** [comment]
Excellent! Thanks!

**Pull request merged**